fuse-products/src/main/java/software/tnb/product/integration/builder/AbstractIntegrationBuilder.java

```diff
@@ -73,6 +73,7 @@ public abstract class AbstractIntegrationBuilder<SELF extends AbstractIntegratio
     private int port = 8080;
 
     private String jvmAgentPath;
+    private List<String> vmArguments;
```

Comment:
`private List<String> vmArguments = new ArrayList<>();` and you don't need to handle null cases

Reply:
ooo yea